lib/ae_mdw/db/util.ex

```diff
@@ -11,31 +12,42 @@ defmodule AeMdw.Db.Util do
   import AeMdw.Sigil
   import AeMdw.Util
 
-  ################################################################################
+  @eot :"$end_of_table"
 
   def read(tab, key),
     do: :mnesia.async_dirty(fn -> Database.read(tab, key) end)
 
   def read!(tab, key),
     do: read(tab, key) |> one!
 
-  def read_tx(txi),
-    do: :mnesia.async_dirty(fn -> Database.read(~t[tx], txi) end)
+  def read_tx(txi) do
+    case RocksDbCF.read_tx(txi) do
+      {:ok, m_tx} -> [m_tx]
+      :not_found -> []
+    end
+  end
 
   def read_tx!(txi),
     do: read_tx(txi) |> one!
 
-  def read_block({_, _} = bi),
-    do: :mnesia.async_dirty(fn -> Database.read(~t[block], bi) end)
+  def read_block({_, _} = bi) do
+    case RocksDbCF.read_block(bi) do
```

Comment:
Direct access to `RocksDbCF` as `Database.dirty_read` is a different operation (behaves differently because previously there was no real transaction). After refactoring the Database removing the Mnesia code, `Database.dirty_read` can have a uniform behavior of reading through the transaction (read uncommitted then if not read committed).